### PR TITLE
Docs - change default log_rotation_size to 1GB

### PR DESCRIPTION
**Pull request by litzell**
- Also clarify how and when log rotation occurs, i.e., log_rotation_size triggers rotation, it's not a maximum file size.
- For log_rotation_age, log is rotated when the current log file reaches specified age, not "once a day".

## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/config_params/guc-list.xml [hunk 1]
```diff
@@ -6250,8 +6250,8 @@
   <topic id="log_rotation_age">
     <title>log_rotation_age</title>
     <body>
-      <p>Determines the maximum lifetime of an individual log file. After this time has elapsed, a
-        new log file will be created. Set to zero to disable time-based creation of new log
+      <p>Determines the lifetime of an individual log file. When this amount of time has elapsed,
```
Comment (lisakowen) [inline gpdb-doc/dita/ref_guide/config_params/guc-list.xml]: *lifetime* makes me think that the original/current log file is deleted after log_rotation_age is reached.   if that's the case, lets state it.  if not, maybe we can say something like "determines the amount of time during which greenplum database writes messages to the active log file.  when this amount of time has elapsed ... "

File: gpdb-doc/dita/best_practices/logfiles.xml [hunk 2]
```diff
@@ -118,9 +118,14 @@
     <p>To search log files on the segment hosts, use the Greenplum <codeph>gplogfilter</codeph>
       utility with <codeph>gpssh</codeph> to connect to segment hosts from the master host. You can
       identify corresponding log entries in segment logs by the <codeph>statement_id</codeph>. </p>
-    <p>The <codeph>log_rotation_age</codeph> configuration parameter specifies when a new log file
-      is automatically created while a database instance is running. By default, a new log file is
-      created every day. </p>
+    <p>Greenplum Database can be configured to rotate database logs based on the size or age of the
```
Comment (dyozie) [inline gpdb-doc/dita/best_practices/logfiles.xml]: or -> and/or

Not 100% sure of the above, but it seems like the config parameters are independent.

File: gpdb-doc/dita/security-guide/topics/Auditing.xml [hunk 3]
```diff
@@ -671,6 +671,17 @@
...
             </row>
+            <row>
+              <entry>
+                <p>log_rotation_size</p>
+              </entry>
+              <entry>0 - INT_MAX/1024 kilobytes</entry>
+              <entry>1048576</entry>
+              <entry>
+                <p>When greater than 0, a new log file is created when this number of kilobytes have
+                  been written to the log. </p>
```
Comment (dyozie) [inline gpdb-doc/dita/security-guide/topics/Auditing.xml]: Might as well add the "Set to zero" sentence here, similar to log_rotation_age.

File: gpdb-doc/dita/best_practices/logfiles.xml (hunk 2 above)
Comment (litzell) [reply gpdb-doc/dita/best_practices/logfiles.xml]: Yes. It was never clear that they work independently.

**Pull request merged**